src/DynamicData/Internal/Rxx.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) 2011-2023 Roland Pheasant. All rights reserved.
+// Roland Pheasant licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for full license information.
+
+#if NET9_0_OR_GREATER
+
+namespace System.Reactive.Linq;
+
+internal static class Rxx
```

Comment:
@JakenVeina / @dwcullop 

This is the main part of the PR.  Can you please consider whether this is correctly implemented

Reply:
Aside from the fact that you're not using the `internal` classes that RX uses for its operators, this looks identical to their `.Synchronize()` operator.